src/test/java/org/springframework/cloud/deployer/spi/cloudfoundry/CloudFoundryAppDeployerTests.java

```diff
@@ -129,12 +152,32 @@ public void shouldNamespaceTheDeploymentIdWhenAGroupIsUsed() throws InterruptedE
 	public void shouldNotNamespaceTheDeploymentIdWhenNoGroupIsUsed() throws InterruptedException, IOException {
 		FileSystemResource resource = new FileSystemResource("src/test/resources/demo-0.0.1-SNAPSHOT.jar");
 
-		String deploymentId = runFullDeployment(new AppDeploymentRequest(
+		cloudFoundryDeploymentProperties.setDomain("wizz.com");
+		cloudFoundryDeploymentProperties.setHost("quik");
+
+		runFullDeployment(new AppDeploymentRequest(
 				new AppDefinition("time", Collections.emptyMap()),
 				resource,
 				Collections.emptyMap()));
 
-		assertThat(deploymentId, equalTo("dataflow-server-time"));
```

Comment:
This is needed for the test case `shouldNotNamespaceTheDeploymentIdWhenNoGroupIsUsed` and I guess we can have a different test case to check these deployment properties override.